cli/filplus.go

```diff
@@ -545,7 +545,7 @@ var filplusRemoveExpiredAllocationsCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() < 1 {
+		if cctx.NArg() != 1 {
```

Comment:
no, this one's correct already, the first hint is in `ArgsUsage` where you see `...`, then further down there's a `Slice()` and then an iteration over however many there are

Reply:
can you elaborate on this? Didn't quite understood it